Database.xcodeproj/project.pbxproj

```diff
@@ -17,6 +17,7 @@
 		2A0CA5751B294C9200ED8781 /* libsqlite3.dylib in Frameworks */ = {isa = PBXBuildFile; fileRef = 2A0CA5741B294C9200ED8781 /* libsqlite3.dylib */; };
```

Comment:
@jenpaff I have renamed the project to Powerup. Here's the link to PR: https://github.com/systers/powerup-iOS/pull/21